ml-fundamentals/README.md

```diff
@@ -19,9 +19,10 @@ For specific aspects of an ML project, refer to the guidelines for different [pr
 ### General
 
 * [ML Fundamentals checklist](ml-fundamentals-checklist.md)
-* [Responsible AI](https://www.microsoft.com/en-us/ai/responsible-ai-resources) (external link)
+* [Responsible AI](https://www.microsoft.com/en-us/ai/responsible-ai-resources)

```

Comment:
I think it's worth mentioning that this is an external link and not a part of this playbook. This README is a TOC for the rest of the ML fundamentals, but some links are to external content. WDYT?

Reply:
We don't do this anywhere else in the playbook so I thought it looked a bit weird to do it here

Reply:
Thanks Tess, I did a quick check in the playbook, and I couldn't see places where the main doc of a fundamental references to external websites, aside from places where it says "References". I think we should either say that it's external (otherwise it's confusing), or put it under a "References" section. WDYT?

Reply:
Not a big fan of adding external since we are not doing this anywhere else so it makes this doc inconsistent with the rest. IMHO will notice that they move to an external resource when they click. But if you feel super strongly about it I can change.

Reply:
I would rather put it under references